### PR TITLE
Implemented a code block to display code snippets in blog posts

### PR DESCRIPTION
**Pull request by silvanocerza**

### CODE REVIEW

File: django-website/blog/fields.py [hunk 1]
```diff
@@ -45,6 +51,32 @@ class YouTubeBlock(StructBlock):
...
+
+    language = ChoiceBlock(choices=LANGUAGE_CHOICES)
+    code_text = TextBlock()
+
+    def render(self, value):
+        src = value['code_text'].strip('\n')
+        lang = value['language']
+
+        lexer = get_lexer_by_name(lang)
+        formatter = get_formatter_by_name(
```
Comment (synasius) [inline django-website/blog/fields.py:74]: set monokai style

File: django-website/wheelie/package.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
   "devDependencies": {
     "gulp": "3.9.0",
     "wheelie": "0.1.3",
-    "wheelie-recipe": "0.2.1"
+    "wheelie-recipe": "0.2.1",
+    "normalize.css": "5.0.0"
```
Comment (rub) [inline django-website/wheelie/package.json:25]: Normalize.css should be a production dependency. Remove normalize.css from devDependencies, then install it by running `npm install --save normalize.css` (or `bower install --save normalize.css` if we're still using Bower);

File: django-website/wheelie/client/scss/core/_reset.scss [hunk 3]
```diff
@@ -1,5 +1,5 @@
 // Reset frameworks
-@import "../../../vendors/normalize-css/normalize";
+@import "../../../node_modules/normalize.css/normalize.css";
```
Comment (rub) [inline django-website/wheelie/client/scss/core/_reset.scss]: Be sure to import the version installed via npm (or Bower if we're still using it).

File: django-website/blog/templates/partials/streamfield.html [hunk 4]
```diff
@@ -28,6 +28,8 @@ <h3 class="article-subtitle">{{ child }}</h3>
         <div class="videoWrapper">
             {{ child }}
         </div>
+    {% elif child.block_type == 'code_snippet' %}
```
Comment (rub) [inline django-website/blog/templates/partials/streamfield.html:31]: Wrap the child into a div with a meaningful class name (e.g. code-snippet).

**Review by rub — CHANGES_REQUESTED** [review]

**Review by rub — APPROVED** [review]

File: django-website/blog/fields.py [hunk 5]
```diff
@@ -45,6 +51,33 @@ class YouTubeBlock(StructBlock):
     identifier = EmbedBlock()
 
 
+class CodeBlock(StructBlock):
+    """ Code Highlighting Block """
```
Comment (Girbons) [inline django-website/blog/fields.py]: Format the docstring like this:
"""
"""
insted of """ """

File: django-website/blog/fields.py [hunk 6]
```diff
@@ -45,6 +51,33 @@ class YouTubeBlock(StructBlock):
...
+        ('css', 'CSS'),
+    )
+
+    language = ChoiceBlock(choices=LANGUAGE_CHOICES)
+    code_text = TextBlock()
+
+    def render(self, value):
+        src = value['code_text'].strip('\n')
+        lang = value['language']
+
```
Comment (Girbons) [inline django-website/blog/fields.py:72]: remove this blank line

File: django-website/blog/fields.py [hunk 7]
```diff
@@ -45,6 +51,33 @@ class YouTubeBlock(StructBlock):
...
+        lexer = get_lexer_by_name(lang)
+        formatter = get_formatter_by_name(
```
Comment (Girbons) [inline django-website/blog/fields.py:74]: insert a blank line here

**Review by Girbons — CHANGES_REQUESTED** [review]

**Pull request merged**